src/sentry/similarity/features.py

```diff
@@ -102,6 +102,9 @@ def record(self, events):
 
         items = []
         for event in events:
+            # TODO: how we index events?
+            if not event.group:
```

Comment:
I think we should check for `event.group_id` to avoid the database query triggered on the access.

Reply:
right. Will change